### PR TITLE
Time integrated reporter value PP

### LINKED ISSUE #99: Time integrated reporter value PP
Implement a PP class to return the time integrated value of a reporter.

### PR DESCRIPTION
**Pull request by somu15**
Implement a PP class to return the time integrated value of a reporter.

Closes #99

### CODE REVIEW

**Comment by rpodgorney** [comment]
@somu15 do you want your PR merged?

**Comment by somu15** [comment]
> @somu15 do you want your PR merged?

Yes, @rpodgorney. Maybe @lynnmunday can you please look at this once?

**Comment by rpodgorney** [comment]
I quickly looked it over, @lynnmunday a second set of eyes would be welcome

File: src/postprocessors/TimeIntegratedReporterValue.C [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "TimeIntegratedReporterValue.h"
+
+registerMooseObject("FalconApp", TimeIntegratedReporterValue);
+
+InputParameters
+TimeIntegratedReporterValue::validParams()
```
Comment (lynnmunday) [inline src/postprocessors/TimeIntegratedReporterValue.C:15]: Shouldn't the time be TimeIntegratedPostProcessor, it isn't a reporter.

**Review by lynnmunday — COMMENTED** [review]
I know this was merged but these need fixed.

**Pull request merged**